src/Objects/PlayerSpawner.gd

```diff
@@ -0,0 +1,18 @@
+extends Position2D
+
+var player_human_scene = preload("res://src/Actors/Player_Human.tscn")
+var player_wasp_scene = preload("res://src/Actors/Player_Wasp.tscn")
+
+
+func _ready():
+    instance_player()
+
+
+func instance_player():
+    var player
+    if GlobalWorld.playerType == "Wasp":
+        player = player_wasp_scene.instance()
+        add_child(player)
+    else:
+        player = player_human_scene.instance()
+        add_child(player)
```

Comment:
Co sądzisz o zrobieniu dwóch if-elsów: if player == Wasp oraz else if player == Human, a w ostatnim bloku else wywalić błąd jeśli dane stworzenie nie jest zaimplementowane? Może w postaci okienka okna dialogowego z errorem albo wpisu do konsoli komunikatu błędu i awaryjnie przypisać Human do playera? Coś takiego:
if gracz == osa: player = wasp
else if gracz == human: player = human
else: wyświetl_błąd; player = human

Reply:
Zastanawiam się nad celem tego errora. 

Będziemy mieli tylko 4 klasy postaci i wychodzę z założenia, że jeżeli playerType przyjmie dowolną wartość, to przypisujemy mu defaultowo Human.

Ale jeżeli uważasz, że to stosowne, to mogę np. wrzucić tam printa do konsoli z informacją o tym, że mieliśmy wartość spoza zakresu.

Reply:
Zmiany dodane.